### PR TITLE
Fix side menu blinking on refresh

### PR DESCRIPTION
**Pull request by IlyaKhD**

### CODE REVIEW

File: templates/react/application/src/utils/patches.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            if (menuOpened) {
+                return;
+            }
+
+            setTimeout(() => {
+                component.setState({ preInitCssFix: false });
+            });
+        }
+    };
+}
```
Comment (ovchinnikov) [inline templates/react/application/src/utils/patches.js]: New line should be here

File: templates/react/application/src/utils/patches.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      height: 100%;
+      display: inline-block;
+      position: absolute;
+      background-color: $base-bg;
+    }
+  
+    .dx-overlay-content {
+      width: $menuMinSize !important;
+    }
+}
```
Comment (ovchinnikov) [inline templates/react/application/src/utils/patches.scss]: And here

File: templates/react/application/src/utils/patches.js (hunk 1 above)
Comment (IlyaKhD) [reply templates/react/application/src/utils/patches.js]: Fixed

File: templates/react/application/src/utils/patches.scss (hunk 2 above)
Comment (IlyaKhD) [reply templates/react/application/src/utils/patches.scss]: Fixed

**Pull request merged**